source/appendices/a-glossary-of-terms.rst

```diff
@@ -690,14 +690,16 @@ Note: The main Code of Practice document takes precedence in the case of any con
      - See Open Access.
      -
 
-   * - OA_Delayed
-     - A COUNTER Access_Type that is reserved for future use and must not be implemented. At the time of the transaction, the content item was available as Open Access because the publisher’s embargo period had expired (delayed Open Access).
+   * - OA_Gold
+     - See Open.
```

Comment:
Open on longer contains a reference to OA_Gold, so maybe the change should be explained here?